pkgs/ffigen/lib/src/config_provider/spec_utils.dart

```diff
@@ -309,6 +309,17 @@ Headers headersExtractor(
 /// error and throws an Exception if not found.
 String findDylibAtDefaultLocations() {
   String? k;
+  final condaEnvPath = Platform.environment['CONDA_PREFIX'] ?? '';
```

Comment:
Maybe separate the new code out to a separate private method that returns a nullable value, and then return here if it's not null.